specification/implementation/index.md

```diff
@@ -3,9 +3,9 @@ title: Implementation of the CSIP
 ---
 # PART II: Implementation of the CSIP
 
-In this part of the document we present an implementation of the requirements and principles discussed in Part I of the CS IP. The implementation consists of two core elements: a fixed physical structure of a CS IP Information Package (Section 4) and the exact use of metadata in METS and PREMIS format (Section 5).
+In this part of the document we present an implementation of the requirements and principles presented in Part I of the specifciation for CS IP. The implementation consists of two core elements: a fixed physical structure of a CS IP Information Package (Section 4) and the exact use of metadata using the "Metadata Encoding & Transmission Standard" (METS) http://www.loc.gov/standards/mets/ and "PREservation Metadata Implementation Strategies" (PREMIS) http://www.loc.gov/standards/premis/ format (Section 5).
 
-As explained above, any implementation is destined to be outdated sooner or later. However, the authors of the CS IP have made their best effort to reuse already available best practices and established core standards, and to carry out intensive discussions within the digital preservation community. All of the above should guarantee that the implementation can be used with only minor updates (for example minor updates to metadata elements) throughout the next few decades.
+As explained earlier, any implementation using a metadata standard is destined to be outdated sooner or later. However, the authors of the CS IP have made their best effort to reuse already available best practices and established core standards, and to carry out intensive discussions within the digital preservation community. All of the above should guarantee that the implementation can be used with only minor updates (for example minor updates to metadata elements) throughout the next few decades.
```

Comment:
is destined to be outdated sooner or later
to
will inevitably become obsolete.

Reply:
Update made.